src/oncall/ui/static/js/oncall.js

```diff
@@ -3484,6 +3484,19 @@ var oncall = {
         });
       });
     }
+  },
+  sanitize: function (string) {
```

Comment:
I can see this sanitize is useful when putting user input into html, but we are forming an url from user input. Have you tested the report's user input with your changes yet? What is the resulting url? Would appreciate if you can post the test results as well.

I am not expert on XSS, but I think url/percent encoding may be more appropriate.

Reply:
changed to use inbuild handlebars functions. Tested in existing names and services as well as names and services that include special characters.